CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # NHS.UK frontend Changelog
 
+## Unreleased
+
+:boom: **Breaking changes**
+
+You must make the following changes when you migrate to this release, or your service might break.
+
+#### Rename legacy component `HTML` params to `html`
+
+If you're using the `card`, `details`, `insetText` or `warningCallout` Nunjucks macros, you need to rename the `HTML` params to `html`.
+
+Before:
+
+```njk
+{{ insetText({
+  HTML: "<p>You'll need to stay away from school, nursery or work until all the spots have crusted over. This is usually 5 days after the spots first appeared.</p>"
+}) }}
+```
+
+After:
+
+```njk
+{{ insetText({
+  html: "<p>You'll need to stay away from school, nursery or work until all the spots have crusted over. This is usually 5 days after the spots first appeared.</p>"
+}) }}
+```
+
+This change was made in [pull request #1259: Review legacy Nunjucks params](https://github.com/nhsuk/nhsuk-frontend/pull/1259).
+
+#### Rename details component `text` params to `summaryText`
```

Comment:
Should we also mention the new feature that this PR adds, which is to enable use of `text` in the details component for the expanded text, where you don't need any HTML (ie a single paragraph)?